osu.Game/OsuGame.cs

```diff
@@ -312,6 +314,8 @@ public void CloseAllOverlays(bool hideToolbar = true)
             foreach (var overlay in focusedOverlays)
                 overlay.Hide();
 
+            ScreenFooter.ActiveOverlay?.Hide();
```

Comment:
do i dare ask what provoked this? i would hope it would never in a million years be required because the code above should already be handling this

i see 7fcbda6cd0cae36763f44301553258b038e0fa6d but like... there's very little context as to what scenario this was talking about

Reply:
There's a test that covered this behaviour as something we wanted.

I believe this one: https://github.com/ppy/osu/blob/b8e6cb9a9c867d903c91206b4e4afa003ca4f7f3/osu.Game.Tests/Visual/Navigation/TestSceneSkinEditorNavigation.cs#L334

Reply:
what I was trying to get at is that I would hope that it's not possible for screen footer to be displaying an overlay that's not already in `focusedOverlays` and therefore would have been hidden by code higher up already - is there something that *does* do that?

Reply:
The one in the test is ModSelectOverlay, which does precisely the thing you'd hope not to see.

I can see if this is salvageable by adding mod select as a global overlay but I don't think it will end well (and will probably lead to the `ScreenFooter` refactor arc)